testing/compatability/docker/example.env

```diff
@@ -0,0 +1,9 @@
+serverUrl=http://host.docker.internal:8066
+serverPollingPort=10943
+serverApiKey=API-123
```

Comment:
Maybe we should default this to the API key that Luke configured in the `Environment -setup` command.

I can't remember the name off the top of my head, but it exists.